libs/novui/src/panda-preset.ts

```diff
@@ -87,9 +84,16 @@ export const novuPandaPreset = definePreset({
         title: TITLE_RECIPE,
         button: BUTTON_RECIPE,
         tabs: TABS_RECIPE,
+        input: INPUT_RECIPE,
       },
     },
   },
+  conditions: {
+    extend: {
+      // Mantine uses *-error.
+      error: '&:is(:error, [data-error=true], [aria-error=true])',
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Custom selector for Panda‏